sql/core/src/main/scala/org/apache/spark/sql/execution/command/resources.scala

```diff
@@ -46,3 +51,56 @@ case class AddFile(path: String) extends RunnableCommand {
     Seq.empty[Row]
   }
 }
+
+/**
+ * Return a list of file paths that are added to resources.
+ * If file paths are provided, return the ones that are added to resources.
+ */
+case class ListFilesCommand(files: Seq[String] = Seq.empty[String]) extends RunnableCommand {
+  override val output: Seq[Attribute] = {
+    val schema = StructType(
+      StructField("Results", StringType, nullable = false) :: Nil)
+    schema.toAttributes
```

Comment:
Thanks! I will update to use `AttributeReference` here. By the way, I check with hive shell, list command does not show an output name. 
```
hive> list files;
/tmp/328231e3-333c-4b7b-9900-87ef634ef86e_resources/test.txt
```